### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [biplobsd/biplobsd.github.io#4] [Announcements]
# index

Next.js, Tailwind, Flutter, flutter_bloc, getx, Selenium, Google Colab, Git, Github, Github Actions, Firebase, Basic Linux command, Visual Studio Code

https://biplobsd.me/

**Comment by undercovernormie** [comment]
Thank you so, so much for your work on the YouTube Subscription Transfer. Myself and my 2006 account's 1000+ subscriptions appreciate a real solution very very much. I tossed you a small donation on Buy Me A Coffee. THANK YOU!!!

**Reply by biplobsd** [reply]
@undercovernormie Thank you so much
